src/main/java/net/ccbluex/liquidbounce/features/module/modules/combat/KillAura.kt

```diff
@@ -179,7 +179,7 @@ object KillAura : Module("KillAura", ModuleCategory.COMBAT, Keyboard.KEY_R) {
         1,
         0..5
     ) { autoBlock != "Off" && smartAutoBlock }
-    private val blockRate by IntegerValue("BlockRate", 100, 1..100) { autoBlock != "Off" && releaseAutoBlock }
+    private val blockRate by IntegerValue("BlockRate", 100, 1..100) { autoBlock !in arrayOf("Off", "Fake") && releaseAutoBlock }
```

Comment:
This is intended, added as a single line change of some commit.

Reply:
> This is intended, added as a single line change of some commit.

You can only see blockrate if releaseAutoBlock is enabled which wasn't possible to enable in the first place except if you swapped to a mode that has the option, enabled it then swapped to Fake, how does that sound?